cmd/vulcan-exposed-db/main.go

```diff
@@ -95,18 +94,25 @@ func exposedDatabases(target string, nmapReport *gonmap.NmapRun, databaseRegex *
 				"Version":  port.Service.Version,
 			}
 			gr.Rows = append(gr.Rows, networkResource)
-
-			e.WithFields(logrus.Fields{"resource": networkResource}).Debug("Resource added")
+			v := exposedVuln
+			v.Resources = []report.ResourcesGroup{gr}
+			v.ID = computeVulnerabilityID(target, v.AffectedResource, port.PortId, port.Protocol, port.Service.Product, port.Service.Version)
```

Comment:
Now there's a new `Vulnerability.Fingerprint` attributed to be used instead of the `ID`. Also we are not adding the target nor the affected resource to calculate the fingerprint now.

Suggested change:
```
v.Fingerprint = helpers.ComputeFingerprint(port.Service.Product, port.Service.Version)
```